Могут быть следующие ситуации:
    - Тест не актуален в течение некоторого времени по ряду причин.
    - Внедряется новая функция, и мы уже добавили тест для этой функции.

В этих ситуациях у нас есть возможность
- пропустить тест:
@pytest.mark.skip
- или не фиксировать ошибку:
@pytest.mark.xfail